config/initializers/status.rb

```diff
@@ -1,3 +1,5 @@
 # frozen_string_literal: true
 
 ENV["CONJUR_VERSION_DISPLAY"] = File.read(File.expand_path("../../VERSION", File.dirname(__FILE__)))
+
+puts "Conjur Version: #{ENV["CONJUR_VERSION_DISPLAY"]}"
```

Comment:
Can you change the message to `Conjur  v#{ENV["CONJUR_VERSION_DISPLAY"]} starting up...`?

It matches the language we use in the authn-client and the secrets-provider.

Reply:
Also, why are you using `puts` and not using the Rails logger? I would move this message to `logs.rb`, add a `CONJ` code to it and print it using the Rails logger.